CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to the QIIME 2 Ecosystem
+
+There are many ways to contribute to the QIIME 2 ecosystem, and you don't have to be a software developer to help out!
+Contributions can come in the form of: new documentation; improvements to existing documentation; assistance with technical support on the [QIIME 2 Forum](https://forum.qiime2.org); new features, bug fixes, or usabililty improvements to existing functionality; development of new plugins or interfaces; providing constructive feedback on how we can improve QIIME 2; and other ways.
```

Comment:
Maybe also add some distinction about users reporting bugs or submitting feature/improvement requests via issues (vs just submitting contributing code for these purposes).